docs/README.md

```diff
@@ -15,6 +15,7 @@ To contribute to the documentation, open a GitHub pull request.
 - Work on your fork's task branches, not the `master` branch.
 - Send pull requests to the `master` branch in this repository.
 - If you want to submit a knowledge base article, see [Submitting a KB Article](#submitting-a-kb-article).
+- Follow the [ScyllaDB Style Guide](https://docs.google.com/document/d/1lyHp1MKdyj0Hh3NprNFvEczA4dFSZIFoukGUvFJb9yE/edit?usp=sharing).
```

Comment:
For the future: we can move this document now into the repository itself.

Reply:
Yes, in the future I'd like to add it to the "Contribute to ScyllaDB" section so that it gets published along with the docs and can easily be referred to (as it should be used for any kind of content, including marketing stuff). I'll send a different PR to make that change.